src/code.cloudfoundry.org/Makefile

```diff
@@ -0,0 +1,8 @@
+.PHONY: clean
+clean:
+	find . -type d -name fakes -exec rm -rf {} \;  || true
+
+.PHONY: generate
+generate:
+	go generate ./...
+	gsed -i '/new(FakeVarzReporter)/i \/\/lint:ignore SA1019 - auto-generated fake will go away when Varz goes away' gorouter/metrics/fakes/fake_varzreporter.go
```

Comment:
Adding a make file just for this one use-case seems a bit odd.`go generate ./...` probably doesn't need one and could we replace the injected linter ignore directive with a staticcheck config that ignores the check for this particular (or all generated) file(s)?

https://staticcheck.dev/docs/configuration/